### PR TITLE
Add support for building dotnet CLI on RHEL.

### PR DESCRIPTION
**Pull request by Sridhar-MS**

### CODE REVIEW

**Comment by davidfowl** [comment]
Looks pretty simple

File: test/EndToEnd/EndToEndTest.cs [hunk 1]
```diff
@@ -244,7 +244,8 @@ private bool IsCentOS()
 
                 if(File.Exists(OSIDFILE))
```
Comment (anurse) [inline test/EndToEnd/EndToEndTest.cs]: Could this not use `PlatformServices.Default.Runtime.OperatingSystem`? That property value is derived from the same file.

**Comment by anurse** [comment]
Minor comment, and needs rebasing, other than those: :shipit:

**Comment by Sridhar-MS** [comment]
@dotnet-bot test OSX Debug Build please.

File: test/EndToEnd/EndToEndTest.cs (hunk 1 above)
Comment (Sridhar-MS) [inline test/EndToEnd/EndToEndTest.cs]: Fixed.

**Pull request merged**